README.md

```diff
@@ -56,13 +56,33 @@ For the purpose of our first example, we'll create a local channel within one pr
 [later examples](./documentation/guides/rust#readme), you'll see that it's just as easy to create
 end-to-end protected channels over multi-hop, multi-protocol transport routes:
 
-1. Install Rust
+1. Deploy Container
+
+    To reduce friction and focus the attention on learning, we recommend the usage of a Docker container for the learning exercise. To learn how to get started with Docker, please visit the [Get Started With Docker](https://docs.docker.com/get-docker/) documentation. Alternatively, you can skip to step 2 if you want to proceed with a non-containerized install.
+
+
+    This command may take a few minutes the first time you invoke it:
+
+    ```
+    docker run --rm -it -e HOST_USER_ID=$(id -u) --name ockam-learn ghcr.io/build-trust/ockam-builder:latest bash
+    ```
+    Upon completion, you will be placed inside the `/work` folder of the container. Next, add a text editior for editing files.
+
+    ```
+    apt update && apt install nano
```

Comment:
I didn't write the original guide but I would say that using `touch` to create the files and not specifying any tool to edit those files is intentional so the user can proceed with their preferred tool (everyone has their own editing preferences after all). Any thoughts on this @mrinalwadhwa ?